examples/oktapam_ad_connection/ad_connection_example.tf

```diff
@@ -13,14 +13,14 @@ resource "oktapam_project" "test-project" {
 }
 
 # Creating gateway is not supported. Get the gateway id using datasource
-data "oktapam_gateway" "gateway-list" {
```

Comment:
James recommended all names be kept consistent with underscores, e.g. `gateway_list`

Reply:
Fixed